src/commands/redis_cmd.cc

```diff
@@ -632,20 +659,23 @@ class CommandPSetEX : public Commander {
   Status Parse(const std::vector<std::string> &args) override {
     auto ttl_ms = ParseInt<int64_t>(args[2], 10);
     if (!ttl_ms) {
-      return Status(Status::RedisParseErr, errValueNotInteger);
+      return {Status::RedisParseErr, errValueNotInteger};
     }
-    if (*ttl_ms <= 0) return Status(Status::RedisParseErr, errInvalidExpireTime);
-    if (*ttl_ms > 0 && *ttl_ms < 1000) {
+
+    if (*ttl_ms <= 0) return {Status::RedisParseErr, errInvalidExpireTime};
+
+    if (*ttl_ms < 1000) {
```

Comment:
Because the condition `*ttl_ms <= 0` was checked a few lines before, here we can omit the `*ttl_ms > 0` check.